packages/generators/src/addon-generator.ts

```diff
@@ -57,7 +58,7 @@ const addonGenerator = (
 
         public async prompting(): Promise<void> {
             if (!this.supportedTemplates.includes(this.template)) {
-                this.utils.logger.warn(`⚠ ${this.template} is not a valid template, please select one from below`);
+                this.utils.logger.log(`${yellow(`⚠ ${this.template} is not a valid template, please select one from below`)}`);
```

Comment:
hm, why we need change warn on log?

Reply:
We're writing to `stdout` for the case of `init`.
https://github.com/webpack/webpack-cli/blob/af6c02b130a35b62b94542680aa7ea41561e77d7/packages/generators/src/init-generator.ts#L61

We follow a similar implementation for the `loader` and `plugin` subcommands. However, using `logger.warn()` writes to `stdin`.
https://github.com/webpack/webpack-cli/blob/af6c02b130a35b62b94542680aa7ea41561e77d7/packages/generators/src/addon-generator.ts#L60

Reply:
writing to stderr is better